infra/base-images/base-runner/dataflow_tracer.py

```diff
@@ -26,9 +26,9 @@
 
 # These can be controlled by the runner in order to change the values without
```

Comment:
Maybe we should just blocklist this file instead of fixing it to pass lint, since:
1. the modifications are slightly more involved for this file
2. There are no tests for this file.
3. Max isn't around to maintain this file anymore.
4. If this file starts failing I assume we will have no idea.

Reply:
Sure, lets blacklist for now.

Reply:
Done.